avm/res/app/container-app/tests/e2e/disable-ingress/main.test.bicep

```diff
@@ -0,0 +1,73 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
```

Comment:
Should probably be updated as the 'title' of the test that 😉

Reply:
Appreciate it 😄 
One small sidenote regarding comments: If possible please only resolve comments that are 'suggestions'. Comments like this one I should re-review to close them and it's just a lot easier to find out which ones if I can resolve them myself 😄